### PR TITLE
Front/ui improvements

### LINKED ISSUE #5998: Visual UI offset on Chrome
**Describe the bug**
There is a missing top padding on button elements but strapi is so great to deserve that

**Steps to reproduce the behavior**
1. Go to any app view on Chrome browser

**Expected behavior**
Label should be verticaly aligned inside the container and verticaly align with the icon if any

**System**
- Strapi version: 3.0.0-beta.20

**Screenshots**
image

### PR DESCRIPTION
**Pull request by virginieky**
#### Description of what you did:
Dashboard UI improvements

Fixes #5998

#### My PR is a:

- [ ] 💥 Breaking change
- [ ] 🐛 Bug fix
- [X] 💅 Enhancement
- [ ] 🚀 New feature

#### Main update on the:

- [X] Admin
- [ ] Documentation
- [ ] Framework
- [X] Plugin

#### Manual testing done on the following databases:

- [X] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

### CODE REVIEW

File: packages/strapi-plugin-users-permissions/admin/src/components/BoundRoute/Components.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ const Verb = styled.div`
   border-radius: 3px;
   color: #ffffff;
   font-size: 12px;
-  font-weight: 700;
+  font-weight: 600;
```
Comment (soupette) [inline packages/strapi-plugin-users-permissions/admin/src/components/BoundRoute/Components.js:42]: You can use the theme here

File: packages/strapi-plugin-users-permissions/admin/src/components/Policies/Components.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ const Header = styled.div`
   margin-bottom: 1.1rem;
   padding-top: 2.8rem;
   font-size: 18px;
-  font-weight: 700;
+  font-weight: 600;
```
Comment (soupette) [inline packages/strapi-plugin-users-permissions/admin/src/components/Policies/Components.js:12]: Same here

File: packages/strapi-provider-upload-local/lib/index.js [hunk 3]
```diff
@@ -31,10 +31,7 @@ module.exports = {
         return new Promise((resolve, reject) => {
           // write file in public/assets folder
           fs.writeFile(
-            path.join(
-              strapi.config.public.path,
-              `/uploads/${file.hash}${file.ext}`
-            ),
+            path.join(strapi.config.public.path, `/uploads/${file.hash}${file.ext}`),
```
Comment (soupette) [inline packages/strapi-provider-upload-local/lib/index.js:34]: Can you update your branche this modification should not be in this PR.

File: packages/strapi-helper-plugin/lib/src/components/Button/StyledButton.js [hunk 4]
```diff
@@ -25,7 +25,7 @@ const primary = css`
   min-width: 14rem;
   font-weight: 500;
   border: 1px solid;
-  background: linear-gradient(315deg, #0097f6 0%, #005eea 100%);
+  background-color: #007eff;
```
[the remark this answers is not in the document]
Comment (virginieky) [reply packages/strapi-helper-plugin/lib/src/components/Button/StyledButton.js:28]: It's still used in the user-permissions plugin and I really think we should migrate all buttons with the Buffet.js one, yes. But it's not what this PR is for. Let's do it in another PR just after merging this one.

File: packages/strapi-helper-plugin/lib/src/index.js [hunk 5]
```diff
@@ -12,7 +12,6 @@ export { default as Button } from './components/Button';
 export { default as ButtonModal } from './components/ButtonModal';
 export { default as CircleButton } from './components/CircleButton';
 export { default as ContainerFluid } from './components/ContainerFluid';
-export { default as EmptyAttributesBlock } from './components/EmptyAttributesBlock';
```
Comment (soupette) [inline packages/strapi-helper-plugin/lib/src/index.js:15]: If it is not used you can delete the component.

File: packages/strapi-helper-plugin/lib/src/index.js [hunk 6]
```diff
@@ -75,7 +74,6 @@ export { default as PopUpWarning } from './components/PopUpWarning';
 export { default as SearchInfo } from './components/SearchInfo';
 export { default as SelectNav } from './components/SelectNav';
 export { default as SelectWrapper } from './components/SelectWrapper';
-export { default as TrashButton } from './components/TrashButton';
```
Comment (soupette) [inline packages/strapi-helper-plugin/lib/src/index.js:78]: Same here

**Pull request merged**